mathics/core/expression.py

```diff
@@ -208,20 +208,9 @@ def __init__(
 
         self._head = head
         self._elements = elements
-
-        # comment mmatera: I found that in certain cases, the elements_properties
-        # passed as the parameter does not matches with those we generate with build:elements_properties.
-        # This is the cause of the error in the test_series.py pytest.
-        # Once it be fixed, we can uncomment the following code and remove the previous line
-        # assert (elements_properties is None or
-        #        self.elements_properties == elements_properties), ("element properties do not match",
-        #                                                           self, (self.elements_properties,
-        #                                                                  elements_properties)
-        #                                                           )
-        # After things are clean we should not run _build_elements_properties. This is done
-        # when needed in self.rewrite_apply_eval()
-        # self.elements_properties = elements_properties
-        self._build_elements_properties()
+        self.elements_properties = elements_properties
+        if elements_properties is None:
+            self._build_elements_properties()
```

Comment:
This is done inside rewrite_apply_eval where it is needed.

Reply:
If I remove lines 212 and 213, all the pytest crashes, probably because we check for elements_properties in another place.
I am investigating it.

Reply:
Now 212 and 213 are not needed anymore.